### PR TITLE
Feature/implement verification protocol

### PR DESCRIPTION
**Pull request by cglotr**
#6

### CODE REVIEW

File: v1.go [hunk 1]
```diff
@@ -97,16 +99,90 @@ func v1(w http.ResponseWriter, r *http.Request) {
...
+				client.VerificationCode = code
+			} else {
+				if client.VerificationCode != code {
+					log.Println("Verification code doesn't match the record.")
+					writeEmptyAction(conn, verificationRequestCodeFailure)
+					break
+				}
+			}
+
+			conn.WriteJSON(Action{
```
Comment (imranariffin) [inline v1.go]: @cglotr `writeEmptyAction` here too? Other than that everything looks good to me.

**Review by imranariffin — DISMISSED** [review]

File: v1.go (hunk 1 above)
Comment (cglotr) [reply v1.go]: Done dc755c3e57b2cbc46428537ca1c48c95680ac168

**Comment by cglotr** [comment]
Resolves #6

**Comment by cglotr** [comment]
Closes #6

**Pull request closed without merging**